playwright/tests/integration/properties/combobox.spec.ts

```diff
@@ -14,8 +14,13 @@ test('default', async ({ page }) => {
   const itemLabel = options.input({ label: 'Item Label' });
   const itemValue = options.input({ label: 'Item Value' });
   const button = options.checkbox({ label: 'Add Dropdown-Button to Combobox' });
+  const behaviour = properties.collapsible('Behaviour');
+  const visible = behaviour.input({ label: 'Visible' });
+  const disable = behaviour.input({ label: 'Disable' });
+  const required = behaviour.input({ label: 'Required' });
+  const requiredMessage = behaviour.input({ label: 'Required Message' });
```

Comment:
instead of copy past this on every element, why not create a new page object, like it was done in the inscription tests?!?

Reply:
Imagine the label 'Visiblity' changes, now you have to change every test, instead of have a page object and only change it there.

Reply:
Yes, you're right 👍 That makes much more sense. I was just a bit confused because, for example, in the general part, the value/label is also tested manually each time without a page object. But I think that's not entirely comparable. I'll go ahead and create a page object for this behavior part.

Reply:
You're right, I could start such changes too, I will not pretend otherwise.
But sometimes it is not worth or it is technically not the same thing. E.g. the label/value is not something which is shared to every component, but the behaviour is.
But of course maybe it would also make sense to to it for those things